homeassistant/components/input_boolean/__init__.py

```diff
@@ -52,6 +55,35 @@ def is_on(hass, entity_id):
     """Set up an input boolean."""
     component = EntityComponent(_LOGGER, DOMAIN, hass)
 
+    entities = await _async_process_config(config)
+    if not entities:
+        return False
+
+    async def reload_service_handler(service_call):
+        """Remove all input booleans and load new ones from config."""
+        conf = await component.async_prepare_reload()
+        if conf is None:
+            return
+        new_entities = await _async_process_config(conf)
+        if new_entities:
+            await component.async_add_entities(new_entities)
+
+    hass.services.async_register(
```

Comment:
Please use `helpers.service.async_register_admin_service`

Reply:
Makes sense. I copy'n'pasted from `automation` platform. Do we want to register `automation.reload` service as an admin service too?

Reply:
Addressed in a31b0eb391bbb7222c9eb9ffaa931d30484908b3